dependencies.gradle

```diff
@@ -1,28 +1,28 @@
 ext {
     versions = [
-            support            : '25.3.1',
-            androidSdk         : [
+            support   : "25.3.1",
```

Comment:
but I like it to be consistent and if I use single then it doesn't make the `versions.support` obvious for the dependencies below 😢  I want the best of both worlds!